<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExcluirMatricula</class>
 <widget class="QWidget" name="ExcluirMatricula">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/recursos/Imagens/remover.png</normaloff>:/recursos/Imagens/remover.png</iconset>
  </property>
  <widget class="QLabel" name="RetornoDeErro">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>128</width>
     <height>109</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Ano:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txbAno">
          <property name="inputMask">
           <string>0000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Semestre:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="txbSemestre">
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly</set>
            </property>
            <property name="inputMask">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>º</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="BtnExcluir">
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>30</y>
     <width>243</width>
     <height>70</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Matricula do aluno: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txbCodAluno">
        <property name="inputMask">
         <string>0000000000000000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Código da disciplina: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txbCodDisciplina">
        <property name="inputMask">
         <string>0000000000000000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txbCodAluno</tabstop>
  <tabstop>txbCodDisciplina</tabstop>
  <tabstop>txbAno</tabstop>
  <tabstop>txbSemestre</tabstop>
  <tabstop>BtnExcluir</tabstop>
 </tabstops>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
